### PR TITLE
V4

### PR DESCRIPTION
**Pull request by zozlak**
# Backward incompatible changes

* Drop `acdh:hasLocationPath` as we don't use it for quite some time already. Triples with that property (e.g. 55k on apollo) can be just removed.

# New properties

* `acdh:hasOrder`
  * rationale: It already popped out a few times that arche-based project services would like to be able to order resources within a collection in a deterministic way and none of existing properties really fits this purpose. This would also allow to extend the Drupal GUI-embedded viewers with controls allowing direct open of the next/previous resource in the collection or to reliably generate iiif manifests for collections.
  * TODO
    * review labels and add German ones
    * decide on `acdh:ordering` value
* Non-linked counterparts to `acdh:hasCoverage` (`acdh:hasNonLinkedCoverage`), `acdh:hasContributor` (`acdh:hasNonLinkedContributor`) and their subproperties (`acdh:hasActor` - `acdh:hasNonLinkedActor`, `acdh:hasSpatialCoverage` - `acdh:hasNonLinkedSpatialCoverage`, `acdh:hasCreator` - `acdh:hasNonLinkedCreator`, `acdh:hasDigitisingAgent` - `acdh:hasNonLinkedDigitisingAgent`, `acdh:hasEditor` - `acdh:hasNonLinkedEditor`, `acdh:hasFunder` - `acdh:hasNonLinkedFunder`, `acdh:hasPrincipalInvestigator` - `acdh:hasNonLinkedPrincipalInvestigator`)
  * rationale: [see here](https://oeawacat.sharepoint.com/:w:/r/sites/ACDH-CH_i_dsie/_layouts/15/doc.aspx?sourcedoc=%7B330e32a0-fb25-40a7-aa41-6c5343833607%7D&action=edit)
  * remarks:
    * Non-linked counterparts were created only for non-compulsory properties as for compulsory ones (`achd:hasContact`, `acdh:hasDepositor`, `acdh:hasHosting`, `acdh:hasLicensor`, `acdh:hasMetadataCreator`, `acdh:hasOwner`, `acdh:hasRightsHolder`) we won't be able to express the cardinality restriction. We can also look at it in a way that compulsory properties so important that we require them to be LOD.
    * Non-linked counterparts don't have `acdh:recommendedClass` annotations but if we think they should, we can add them.
    * Non-linked counterparts don't have `owl:inverseOf` annotations because they are datatype properties.
  * TODO
    * decide on naming. I've chosen `acdh:hasNonLinked{originalProperty}` because we already had `acdh:hasNonLinkedIdentifier` but I'm not personally attached
    * review labels and add German ones
    * decide on `acdh:ordering` values

### CODE REVIEW

**Review by csae8092 — APPROVED** [review]

**Comment by bellerophons-pegasus** [comment]
I do agree with introducing acdh:hasOrder and dropping acdh:hasLocationPath

The introduction of non-linked counterparts to `acdh:hasCoverage` (acdh:hasNonLinkedCoverage), `acdh:hasContributor`(acdh:hasNonLinkedContributor) and their subproperties raises many red flags and from the notes I can't really see why we really need them...

**Comment by bellerophons-pegasus** [comment]
Could you maybe please remind me on how the folder structure of downloaded collections is now determined without `acdh:hasLocationPath`?

**Review by bellerophons-pegasus — CHANGES_REQUESTED** [review]
Please think again about introducing the new non-linked properties

**Comment by zozlak** [comment]
> Could you maybe please remind me on how the folder structure of downloaded collections is now determined without `acdh:hasLocationPath`?

It's based on `acdh:isPartOf` now (see [#13481](https://redmine.acdh.oeaw.ac.at/issues/13481#note-21))

**Comment by bellerophons-pegasus** [comment]
@zozlak and I talked and came to this conclusion:

Introduce the following properties:
* `acdh:hasOrder` - all fine (we didn't talk about it, but for the sake of completeness is included in this list
* `acdh:hasNonLinkedActor`

Why not the others as well? Because it seems that most messy named entities (Agents) are coming in via `acdh:hasActor` and also the other properties more often tend to denote living people (or more current organisations) where we can ask depositors to provide an ORCID or another identifier.

As for `acdh:hasCoverage`, especially `acdh:hasSpatialCoverage`: see [Redmine #19100](https://redmine.acdh.oeaw.ac.at/issues/19100)

Thus, remove `acdh:hasNonLinkedCoverage`, `acdh:hasNonLinkedSpatialCoverage`,  `acdh:hasNonLinkedContributor`, `acdh:hasNonLinkedCreator`, `acdh:hasNonLinkedDigitisingAgent`, `acdh:hasNonLinkedEditor`, `acdh:hasNonLinkedFunder`,  and `acdh:hasNonLinkedPrincipalInvestigator`

**Comment by bellerophons-pegasus** [comment]
For `acdh:hasNonLinkedActor` please include

```
<rdfs:comment xml:lang="en">Any Person or Organisation (B) which the given Project, Collection or Resource (A) mentions or is about that has no identifier any of the authority files supported by the repository.</rdfs:comment>
<rdfs:comment xml:lang="de">Jede Person oder Organisation (B), die in der vorliegenden Ressource, der Sammlung oder dem Projekt (A) erwähnt wird oder mit der sie sich befasst und die keinen Identifikator in einer der vom Repositorium unterstützten  Normdateien hat.</rdfs:comment>
<skos:altLabel xml:lang="en">Has non-linked Actor</skos:altLabel>
<skos:altLabel xml:lang="de">Hat Akteur ohne Link</skos:altLabel>
```

For `acdh:hasOrder` please include

```
<acdh:ordering rdf:datatype="http://www.w3.org/2001/XMLSchema#integer">176</acdh:ordering>
<rdfs:comment xml:lang="en">Indicates the order of a Resource or Collection within a containing Collection.</rdfs:comment>
        <rdfs:comment xml:lang="de">Gibt die Reihenfolge einer Ressource oder Sammlung innerhalb einer Sammlung an.</rdfs:comment>
        <skos:altLabel xml:lang="de">Sortiernummer</skos:altLabel>
        <skos:altLabel xml:lang="en">Order number</skos:altLabel>
```

Also: should `acdh:hasOrder` have a default value when not provided? Or be filled with some kind of automation?

**Review by bellerophons-pegasus — APPROVED** [review]

**Comment by zozlak** [comment]
@bellerophons-pegasus I have a doubt regarding the `acdh:hasCustomIiifManifest` name. As raised in https://redmine.acdh.oeaw.ac.at/issues/22279 it is very plausible we will need/want to mark all collections for which a IIIF manifest should be generated with a dedicated metadata property. For such a usage a `hasCustomIiifManifest` sounds a little strange and I would rather avoid having two separate properties for dealing with IIIF manifests (one for a specific and one for an automatically generated). For that reason I would opt for calling this property just an `acdh:hasIiifManifest`.

My second doubt is about the `acdh:hasOrder`. After thinking a little about it I'm not sure if it's better to introduce the `acdh:hasOrder` or maybe `acdh:hasNext[Object]`. I will write down my thoughts about it in the https://redmine.acdh.oeaw.ac.at/issues/22435 in a moment.

File: acdh-schema.owl [hunk 1]
```diff
@@ -212,14 +242,26 @@ Kardinalität von acdh:hasCategory für acdh:BinaryContent gändert zu 0-n, um m
 
     <owl:AnnotationProperty rdf:about="https://vocabs.acdh.oeaw.ac.at/schema#defaultValue">
         <rdfs:comment xml:lang="de">Gibt den Standardwert einer Eigenschaft an. Der Standardwert wird verwendet, wenn diese Eigenschaft in einer Klasse verwendet wird, aber kein Wert angegeben wurde.</rdfs:comment>
-        <rdfs:comment xml:lang="en">Indicates a default property value. The default value is assigned when a class matches property&apos;s domain but no property value has been provided.</rdfs:comment>
+        <rdfs:comment xml:lang="en">Indicates a default property value. The default value is assigned when a class matches property&amp;apos;s domain but no property value has been provided.</rdfs:comment>
```
Comment (carlonim) [inline acdh-schema.owl:245]: I think there is a small typo here, `&amp;apos;` should be corrected to `&apos;`

File: acdh-schema.owl [hunk 2]
```diff
@@ -1729,6 +1788,22 @@ https://www.iana.org/assignments/character-sets/character-sets.xhtml</rdfs:comme
...
+        <rdfs:domain rdf:resource="https://vocabs.acdh.oeaw.ac.at/schema#BinaryContent"/>
+        <rdfs:range rdf:resource="http://www.w3.org/2001/XMLSchema#anyURI"/>
+        <rdfs:comment xml:lang="de">Gibt das IIIF-Manifest (B) an, das für die Darstellung der Ressource (A) verwendet werden soll. Der Wert ist entweder die URL des zu verwendenden IIIF-Manifests (externe URLs sind möglich, wir empfehlen jedoch Ressourcen aus dem Repositorium) oder, wenn das IIIF-Manifest automatisch auf der Grundlage der Metadaten der Ressource generiert werden soll, die URI des Dissemination-Services: https://id.acdh.oeaw.ac.at/dissemination/iiifmanifest.</rdfs:comment>
+        <rdfs:comment xml:lang="en">Indicates the IIIF manifest (B) to be used for the presentation of the Resource (A). The value is either an URL of the IIIF manifest to be used (we recommend it to be another Resource in the repository but it can also be external)  or, if the IIIF manifest should be generated automatically based on the Resoure&apos;s metadata, the URI of the dissemination service: https://id.acdh.oeaw.ac.at/dissemination/iiifmanifest.</rdfs:comment>
```
Comment (carlonim) [inline acdh-schema.owl:1798]: Two small typos:
1. delete double white space between `external)` and `or`
2. correct `Resoure` to `Resource`

File: acdh-schema.owl [hunk 3]
```diff
@@ -161,6 +161,36 @@ Domain für acdh:hasFilename geändert (um acdh:Collection und acdh:TopCollectio
...
+Renamed to ARCHE Schema (was ACDH Schema before)
+Introduced annotation property acdh:exampleValue to allow for the inclusion of example values.
+Introduced acdh:hasNextItem with domain acdh:RepoObject.
+Introduced acdh:hasIiifManifest with domain acdh:BinaryContent.
+Introduced acdh:hasRightsInfomation with domain acdh:RepoObject.
+Expanded description for acdh:hasOaiSet to include visibility in OpenAIRE.
+
+Dropped acdh:hasLocationPath as it wasn&apos;t technically required anymore.
+
+Added acdh:automatedFill &apos;true&apos; to acdh:disseminationServices.
```
Comment (carlonim) [inline acdh-schema.owl:174]: `acdh:automatedFill` for `acdh:disseminationServices` as well as for `acdh:technical` was already introduced in [version 3.1.2](https://github.com/acdh-oeaw/arche-schema/releases/tag/v3.1.2), so we could add a new versionInfo related to 3.1.2 and mention this there.

**Review by carlonim — APPROVED** [review]
Everything seems ok from a formal point of view. I just added some comments regarding small typos:
- on versionInfo: https://github.com/acdh-oeaw/arche-schema/pull/20/files#r1448722722
- on line 245: https://github.com/acdh-oeaw/arche-schema/pull/20/files#r1448718495
- on line 1798: https://github.com/acdh-oeaw/arche-schema/pull/20/files#r1448720161

File: acdh-schema.owl (hunk 3 above)
Comment (bellerophons-pegasus) [reply acdh-schema.owl:174]: Since it was a minor change anyway I think it is okay to leave it as it is

**Review by bellerophons-pegasus — APPROVED** [review]
We'll leave the version info as it is. Otherwise thank you for noting the typos.

**Pull request merged**